letsencrypt/storage.py

```diff
@@ -439,6 +439,18 @@ def names(self, version=None):
         with open(target) as f:
             return crypto_util.get_sans_from_cert(f.read())
 
+    def autodeployment_is_enabled(self):
```

Comment:
Coverage was coming from `test_should_autodeploy`, but it doesn't hurt to have explicit unit tests.